src/poetry/core/utils/_compat.py

```diff
@@ -3,7 +3,6 @@
 from typing import List
 
 
-PY36 = sys.version_info >= (3, 6)
 PY37 = sys.version_info >= (3, 7)
```

Comment:
we should be able to drop `PY37` and any references (I see a couple in the tests)

Reply:
ec61d70